TaskWithStrlen/TaskWithStrlen.cpp

```diff
@@ -0,0 +1,34 @@
+﻿#include <iostream>
+
+int strLength(char str[]) {
+	int count = 0;
+	while (str[count] != '\0')
+	{
+		count++;
+	}
+
+	return count;
+}
+
+int main()
+{
+	char str[] = "Hello World";
+
+	std::cout << strLength(str) << std::endl;
+
+	std::cout << strlen(str) << std::endl;
+
+	for (int i = 0; i < sizeof(str); i++)
```

Comment:
Iterate through the str is meaningless because you know it's a copy of a literal, so you can firmly rely on the compiler the string ends in str[sizeof(str) - 1), str[strlen(str)].
I'm proposing to change. Either:
str[sizeof(str) - 1) = 'd';
Or:
str[strlen(str)] = 'd';

Reply:
Fixed